src/NUnitConsole/nunit3-console.tests/CommandLineTests.cs

```diff
@@ -513,7 +520,13 @@ public void ResultOptionWithoutFileNameIsInvalid()
         [Test]
         public void ResultOptionMayBeRepeated()
         {
-            const string transformFile = "TextSummary.xslt";
+#if !NETCOREAPP1_1
+            var dir = Path.GetDirectoryName(typeof(CommandLineTests).Assembly.Location);
+#else
+            var dir = Path.GetDirectoryName(typeof(CommandLineTests).GetTypeInfo().Assembly.Location);
+#endif
```

Comment:
This and the other conditional places should use `TestContext.TestDirectory`, which I believe already has the necessary conditional code. If it doesn't it should be added there rather than in multiple places.